### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: Functions/It.ps1 [hunk 1]
```diff
@@ -16,7 +16,7 @@ In addition to using your own logic to test expectations and throw exceptions,
 you may also use Pester's Should command to perform assertions in plain language.
 
 .PARAMETER Name
-An expressive phsae describing the expected test outcome.
+An expressive phase describing the expected test outcome.
```
Comment (jsoref) [inline Functions/It.ps1]: Perhaps it should be `phrase`?

Comment (dlwyatt) [inline Functions/It.ps1]: Yep, "phase" makes no sense.  Should be "phrase".

**Comment by dlwyatt** [comment]
Please rebase these into a single commit before we merge.  Thanks!

**Comment by alexandear** [comment]
@dlwyatt why this PR is not merged?

**Comment by nohwnd** [comment]
@alexandear it is now. :)

**Pull request merged**